src/Storage/FileStorage.php

```diff
@@ -62,7 +63,8 @@ public function read(string $type, ?string $id = null): array
         foreach ($dataFiles as $file) {
             $dir = dirname($file);
             $id = substr($dir, strlen(dirname($file, 2)) + 1);
-            $data[$id] = Json::decode(file_get_contents($file));
+            $content = file_get_contents($file);
```

Comment:
What if it return `false`?

Reply:
This behavior is not changed.

We can improve it in separate PR. What behavior do you expect here in this case?